docs: align config descriptions (#1168)
elastic/apm-agent-dotnet: docs/configuration.asciidoc

When recording, the agent captures HTTP requests, tracks errors, and collects and sends metrics.
When not recording, the agent works as a noop, not collecting data and not communicating with the APM server,
except for polling the central configuration endpoint.

> As this is a reversible switch, agent threads are not being killed when inactivated, but they will be

Reviewer: ```suggestion
As this is a reversible switch, agent threads are not killed when inactivated, but they will be
```

[[config-server-url]]
==== `ServerUrl`

The URL for your APM Server. The URL must be fully qualified, including protocol (`http` or `https`) and port.

> IMPORTANT: Use of `ServerUrls` is deprecated. Use `ServerUrl`.

Reviewer: ++ thanks!

| `http://localhost:8200` | String
|============

[float]
[[config-secret-token]]
==== `SecretToken`

> This string is used to ensure that only your agents can send data to your APM server.

Reviewer: ```suggestion
A string used to ensure that only your agents can send data to your APM server.
```

| `<none>`                | String
|============

[float]
[[config-api-key]]
==== `ApiKey` (added[1.4])

> This base64-encoded string is used to ensure that only your agents can send data to your APM server.

Reviewer: ```suggestion
A base64-encoded string used to ensure that only your agents can send data to your APM server.
```

Setting `MaxBatchEventCount` to 0 or a negative value is invalid and
in that case the Agent will use the default value instead.

[options="header"]
|============
| Environment variable name | IConfiguration or Web.config key
| `ELASTIC_APM_MAX_BATCH_EVENT_COUNT` | `ElasticApm:MaxBatchEventCount`
|============

[options="header"]
|============
| Default                 | Type
| 10                      | Integer
|============

[float]
[[config-max-queue-event-count]]
==== `MaxQueueEventCount` (added[1.1])

> The maximal number of events to hold in the queue as candidates to be sent.

Reviewer: ```suggestion
The maximum number of events to hold in the queue as candidates to be sent.
```
